backend/database/seeders/fixes/AccountFixSeeder.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Database\Seeders\Fixes;
+
+use Illuminate\Database\Seeder;
+
+use App\Models\Account;
+use App\Traits\GenerateUniqueCode;
+
+class AccountFixSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $accounts = Account::where('account_number', null)->get();
+        foreach ($accounts as $account) {
+            $account->account_number = GenerateUniqueCode::generate();
```

Comment:
my code changes for this file

`$account->account_number = $account->generateUniqueCode();`